src/trunc.js

```diff
@@ -0,0 +1,34 @@
+import { bind } from 'ramda';
+import curry1 from 'ramda/src/internal/_curry1';
+
+import polyfill from './internal/polyfills/Math.trunc';
+import isFunction from './isFunction';
+
+export const truncPolyfill = curry1(polyfill);
+
+/**
+ * Returns the integer part of a number by removing any fractional digits.
+ *
+ * @func trunc
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.15.0|v2.15.0}
+ * @category Math
+ * @sig Number | String -> Number
+ * @param {number|string} number The number to trunc
+ * @return {number} The integer part of the given number
+ * @example
+ *
+ * RA.trunc(13.37); //=> 13
+ * RA.trunc(42.84); //=> 42
+ * RA.trunc(0.123); //=>  0
+ * RA.trunc(-0.123); //=> -0
+ * RA.trunc('-1.123'); //=> -1
+ * RA.trunc(NaN); //=> NaN
+ * RA.trunc('foo'); //=> NaN
+ */
+
+const trunc = isFunction(Math.trunc)
+  ? curry1(bind(Math.trunc, Math))
+  : truncPolyfill;
```

Comment:
Not sure I understand your comment. Can you explain what you mean in more details pls ?

Reply:
I think it should be

```
const trunc = isFunction(Math.trunc)
  ? curry1(bind(Math.trunc, Math))
  : curry1(truncPolyfill);
```

Reply:
truncPolyfill is alrady curried and exported [here](https://github.com/char0n/ramda-adjunct/pull/821/files#diff-2332de2526929ea292eb2828f3358c2dR7).  It need to be like that so that I can import polyfill in test separatelly and test it in isolation. The same pattern is just being reporuduced in these new PRs from original [repeatStr](https://github.com/char0n/ramda-adjunct/pull/714) PR.

Reply:
see now